server/emails/mailer.tsx

```diff
@@ -180,6 +188,8 @@ export class Mailer {
           `Preview Url: ${nodemailer.getTestMessageUrl(info)}`
         );
       }
+
+      return info?.messageId;
```

Comment:
Ah, I didn't realize the `Message-ID` is something that comes back from the email server originally, I thought it was something we could provide.

Reply:
We can provide it as well (just need to ensure it is unique) - thought we could let the the library handle it.
Perhaps `<notification-id@outline.com>`?